commit a9f032312f8c1254b00840dd084e241c964897ea
Author: Balló György <ballogyor@gmail.com>
Date:   Tue Oct 7 21:19:52 2014 +0200

    Set lxpolkit by default, so lxsession will autostart it on login.

diff --git a/lxsession/settings.vala b/lxsession/settings.vala
index 399deb2..c31f49d 100644
--- a/lxsession/settings.vala
+++ b/lxsession/settings.vala
@@ -252,6 +252,7 @@ namespace Lxsession
             /* Keep old behavior for autostarted application if this option is not specify */
             set_generic_default("Session", "disable_autostart", null, "string", "no");
 
+            set_generic_default("Session", "polkit", "command", "string", "lxpolkit");
             set_generic_default("Session", "clipboard", "command", "string", "lxclipboard");
             set_generic_default("Session", "xsettings_manager", "command", "string", "build-in");
             set_generic_default("Session", "proxy_manager", "command", "string", "build-in");
